src/config/settings.py

```diff
@@ -145,14 +146,38 @@
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.2/howto/static-files/
-
-STATIC_URL = "static/"
```

Comment:
What is the reason we rename our directory with static from statc/ to staticfiles/?